fstest/testserver/init.d/TestWebdavOwncloud

```diff
@@ -12,7 +12,7 @@ start() {
     docker run --rm -d --name $NAME \
            -e "OWNCLOUD_DOMAIN=${OWNCLOUD_DOMAIN}" \
            -e "OWNCLOUD_DB_TYPE=sqlite" \
-           -e "OWNCLOUD_DB_NAME=oowncloud.db" \
+           -e "OWNCLOUD_DB_NAME=owncloud.db" \
```

Comment:
I presume this is ok. it's possible it's intentional, in which case it'd be helpful to understand why...

Reply:
I think it is an accident, so your fix is good